vexim/site.php

```diff
@@ -12,6 +12,11 @@
   if (!isset($_POST['searchfor'])) {
     $_POST['searchfor'] = '';
   }
+  if(isset($_SESSION['pw_enforce'])) {
+    header ("Location: sitepassword.php");
+    die;
+  }
+
```

Comment:
I think we need similar precautions in `siteadd.php`, `siteaddsubmit.php`, `sitechange.php`, `sitechangesubmit.php` and `sitedelete.php`, don't we?